src/pages/Faucets/faucets.json

```diff
@@ -47,80 +47,119 @@
   {
     "alias": "Nord Banano Faucet",
     "account": "ban_1sexmes51h4udqk58dt3rbbpkssbircc7giryewd59q3t5fmfe9crbewjk5y",
-    "link": "https://nord.valejo.net/"
+    "link": "https://nord.valejo.net"
   },
   {
     "alias": "BananoFaucet.club",
     "account": "ban_3rzg5w93ipik4aie3uzafetsfa6dnqdrrsmw9c9dussge7eb5s45xmh4o4pa",
-    "link": "https://www.bananofaucet.club/"
+    "link": "https://www.bananofaucet.club"
   },
   {
     "alias": "nano2go",
     "account": "ban_14yf998msf8zh1du3zchxbetqqtzjjczp84ntnku1atwttwo558us8ewo16z",
-    "link": "https://nano2go.herokuapp.com/"
+    "link": "https://nano2go.herokuapp.com"
   },
   {
     "alias": "BananoForest Faucet",
     "account": "ban_3sinkoff1yj9z5fougwao1gbjtsmb98u1j5p9kcrndqcc4irdxgzsjbem96e",
-    "link": "https://faucet.bananoforest.com/"
+    "link": "https://faucet.bananoforest.com"
   },
   {
     "alias": "Pronoun Faucet",
     "account": "ban_3eeq61ea33jdds5x37otx51esi8wsnxxjc8spjajyq7pj8h3nodkd19pride",
-    "link": "https://banpridefaucet.repl.co/"
+    "link": "https://banpridefaucet.repl.co"
   },
   {
     "alias": "Gorilla Nation",
     "account": "ban_1gori11a7fz3tee6aydqxcehttzbuk93pjsctanfkgqmesa3dmo1cyw89xex",
-    "link": "https://gorillanation.cc/"
+    "link": "https://gorillanation.cc"
   },
   {
     "alias": "Barrel O' Bananos",
     "account": "ban_1barre1777qqdcg86788tk6ojy9jmkyb8ridreezbgkhr7btnoqcntejrxhf",
-    "link": "https://www.devinmontes.com/"
+    "link": "https://www.devinmontes.com"
   },
   {
     "alias": "GoBanano",
     "account": "ban_3w91yp9ufjhpzt98xzddfmnoo8bm51ic3fp3mmx4mqbocqqde7dii1w8g4b6",
-    "link": "https://gobanano.com/"
+    "link": "https://gobanano.com"
   },
   {
     "alias": "BananoTime! Giveaway Bot",
     "account": "ban_3sbqs38uoi9z34z5xeof5y4qjfc41n9rgr4hhhcu5zh5two9xrh4she1n8hs"
+  },
+  {
+    "alias": "EARNS.CC Banano Faucet",
+    "account": "ban_3ze6byhismfpj8jhuda6fq1a7nkniq7t5soet9k58y3egt3rco66j69fdksg",
+    "link": "https://ban.earns.cc"
+  },
+  {
+    "alias": "Banoboto's Future Faucet",
+    "account": "ban_3jzi3modbcrfq7gds5nmudstw3kghndqb1k48twhqxds3ytyj4k7cf79q5ij",
+    "link": "https://faucet.banoboto.repl.co"
+  },
+  {
+    "alias": "Try Banano!",
+    "account": "ban_3x4ya94gh6b8to54su8xy7nwys9n1mn5ydjn6sq4smq9fx9qi4qw1bguab7o",
+    "link": "https://trybanano.com"
+  },
+  {
+    "alias": "i can haz nano",
+    "account": "ban_1monkecrqoqr6j6qzhtd9i8x49ujdnoqt7ramt9jmhd543icsrx5accoqtd5",
+    "link": "https://icanhaznano.monke42.tk"
+  },
+  {
+    "alias": "Yet Another BAN Faucet",
+    "account": "ban_1monkecrqoqr6j6qzhtd9i8x49ujdnoqt7ramt9jmhd543icsrx5accoqtd5",
+    "link": "https://yet-another-ban-faucet.herokuapp.com"
+  },
+  {
+    "alias": "CSquared's Banano Dispensary",
+    "account": "ban_3jyqzypmcn94dmyp7eb3k85sug1568xwehzx738o5jniaxaf1jpxdakjz96r",
+    "link": "https://getban.csquared.nz"
+  },
+  {
+    "alias": "Wise Tribe Faucet",
+    "link": "https://faucet.wisetribe.digital"
```

Comment:
is there an `account` for this one? iirc it's required in the code for linking

Reply:
I don't know either. It looks like the faucet swaps PAW for BAN. I'm not sure if they set receiver's address as a swap address or they have a wallet to send out Banano.

Reply:
This is unclear, I think we can discard this faucet for now.